lms/static/js/student_account/views/LoginView.js

```diff
@@ -19,12 +21,15 @@
                 formType: 'login',
                 requiredStr: '',
                 submitButton: '.js-login',
+                formSuccessTpl: formSuccessTpl,
+                formStatusTpl: formStatusTpl,
+                defaultFormErrorsTitle: gettext("We couldn't sign you in."),
```

Comment:
Nitpick: single quotes should be used instead of double wherever possible in keeping with the coding style of the repo.